### PR TITLE
LPS-192792 | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-192792

cc: @patriciajeanperez

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -655,15 +655,6 @@ definition {
 			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_ASSET_TITLE");
 	}
 
-	@description = "Verify that in Update Due Date modal, the user can expand the comment box"
```
Comment (kristinonias) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:658]: These tests were either discarded or being covered by a different test. I removed them from the test cases.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowCalendar.testcase [hunk 2]
```diff
@@ -525,15 +525,6 @@ definition {
 		CalendarEvent.viewStatusViaDetails(status = "Pending");
 	}
 
-	@description = "Verify that the Calendar Event status changes when approved"
```
Comment (kristinonias) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowCalendar.testcase:528]: [LRQA-68597](https://liferay.atlassian.net/browse/LRQA-68597) This is already covered in WorkflowCalendar#CanApproveCalendarEventOnSingleApprover

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase (hunk 1 above)
Comment (kristinonias) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:658]: [LRQA-68860](https://liferay.atlassian.net/browse/LRQA-68860) This test was removed from the test map

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSourceReact.testcase [hunk 3]
```diff
@@ -43,15 +43,6 @@ definition {
 
 	}
 
-	@description = "Verify that the user is able to import a XML file"
```
Comment (kristinonias) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSourceReact.testcase:46]: [LRQA-74659](https://liferay.atlassian.net/browse/LRQA-74659)  This test is already covered in several other tests

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletedItems.testcase [hunk 4]
```diff
@@ -366,22 +366,4 @@ definition {
...
-	@description = "LPS-86650 - Verify that the unexpected error state appear when unexpected error happen"
-	@ignore = "Test Stub"
-	@priority = 2
-	test UnexpectedErrorState {
-
-		// TODO LRQA-67648 UnexpectedErrorState pending implementation
-
-	}
-
-	@description = "LPS-86650 - Verify that, for numbers greater than 3 digits, use abbreviation standards considering localization specifications (example for English: replace 1,220 for 1.2K)"
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletedItems.testcase:378]: [LRQA-67645](https://liferay.atlassian.net/browse/LRQA-67645) This test is already covered in BE/FE tests

**Comment by kristinonias** [comment]
@patriciajeanperez I updated this PR to have the tickets linked for the test stubs and the reason it was removed from the testcase.

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**